assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -184,11 +185,12 @@ function statusPoll(
   dispatch: Function,
   csrf: CsrfState,
   referrerAcquisitionData: ReferrerAcquisitionData,
+  paymentMethod: PaymentMethod,
 ) {
 
   if (pollCount >= MAX_POLLS) {
-    const url: string = addQueryParamToURL(routes.recurringContribPending, 'INTCMP', referrerAcquisitionData.campaignCode);
-    window.location.assign(url);
+    dispatch(checkoutPending(paymentMethod));
+    return null;
```

Comment:
Before we were returning always `undefined` and now for one of the function's' flow we return `null`. I prefer `null` than `undefined` but we should be consistent as well. we could replace this `return null;` with `return;`